src/olympia/reviewers/management/commands/review_reports_sql/addon/breakdown.sql

```diff
@@ -2,30 +2,50 @@ SELECT 'All Reviewers' AS `Group`,
        IFNULL(FORMAT(SUM(aa.weight), 0), '-') AS `Total Risk`,
        IFNULL(FORMAT(AVG(aa.weight), 2), '-') AS `Average Risk`,
        FORMAT(COUNT(*), 0) AS `Add-ons Reviewed`
-FROM `reviewer_scores` `rs`
-LEFT JOIN `editors_autoapprovalsummary` `aa` ON `aa`.`version_id` = `rs`.`version_id`
-JOIN `users` `u` ON `u`.`id` = `rs`.`user_id`
-WHERE DATE(`rs`.`created`) BETWEEN @WEEK_BEGIN AND @WEEK_END
-  AND `u`.`deleted` = 0
+FROM (
+  /* NOTE: This works to reduce reviews that acted on multiple reviews to a single review
+     because we set created to be the exact same datetime. */
+  SELECT `created`, `user_id`, `action`, MAX(`id`) AS `id`
+  FROM `log_activity`
+  WHERE DATE(`created`) BETWEEN @WEEK_BEGIN AND @WEEK_END
+  /* The type of review, see constants/activity.py */
+  AND FIND_IN_SET(`action`, @ACTIVITY_ID_LIST) > 0
+  GROUP BY `created`, `user_id`, `action`
```

Comment:
Wouldn't this fail if a user submits two separate reviews at the same time?

Reply:
if they were submitted at _exactly_ the same time (i.e. same millisecond), yes.  Seemed like an edge case that wouldn't affect the numbers in a significant way.

Reply:
Sounds ok. I wanted to make sure I got it right and we are aware of it.